lib/modules/datasource/github-release-attachments/index.ts

```diff
@@ -0,0 +1,252 @@
+// TODO: types (#7154)
```

Comment:
```suggestion
```
seems to be resolved

Reply:
I've kept this untouched from the original source, but will remove while being at it.